app/src/main/java/com/example/weather_app/ForecastViewModel.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.weather_app
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import javax.inject.Inject
+
+
+class ForecastViewModel @Inject constructor(private val service: Api) : ViewModel()
+{
+    val forecastConditions: MutableLiveData<Forecast> = MutableLiveData()
+
+
+    fun loadData() = runBlocking {
+        launch {
+            forecastConditions.value = service.getForecast("98178")
+        }
+    }
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+    /*
```

Comment:
Lot's of extra new lines above and commented out code shouldn't go into your git repo.